src/Files.App/Dialogs/CreateShortcutDialog.xaml

```diff
@@ -0,0 +1,61 @@
+﻿<ContentDialog
+	x:Class="Files.App.Dialogs.CreateShortcutDialog"
+	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	xmlns:helpers="using:Files.App.Helpers"
+	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	Title="{helpers:ResourceString Name=NewShortcutDialogTitle}"
+	CornerRadius="{StaticResource OverlayCornerRadius}"
+	DefaultButton="Primary"
+	PrimaryButtonClick="ContentDialog_PrimaryButtonClick"
+	PrimaryButtonText="{helpers:ResourceString Name=Continue}"
+	RequestedTheme="{x:Bind helpers:ThemeHelper.RootTheme}"
+	SecondaryButtonText="{helpers:ResourceString Name=Cancel}"
+	Style="{StaticResource DefaultContentDialogStyle}"
+	mc:Ignorable="d">
+
+	<Border MinWidth="400">
+		<StackPanel Orientation="Vertical" Spacing="12">
+			<TextBlock 
```

Comment:
If there's already a `Grid`, we should be able to remove the `StackPanel` and add an extra row to the `Grid`.